api/client/stats_helpers.go

```diff
@@ -150,7 +154,7 @@ func (s *containerStats) Collect(cli client.APIClient, streamStats bool, waitFir
 				s.mu.Lock()
 				s.err = err
 				s.mu.Unlock()
-				return
+				continue
```

Comment:
Technically we don't need to set everything to zero any more, but that's just noise - thanks for the update!